website/docs/r/vapp_vm.html.markdown

```diff
@@ -175,3 +175,11 @@ its parameters do force recreation of VMs!
   * `ip_allocation_mode=MANUAL` - **`ip`** value must be valid IP address from a subnet defined in `static pool` for network.
 
   * `ip_allocation_mode=NONE` - **`ip`** field can be omitted or set to an empty string "". Empty string may be useful when doing HCL variable interpolation.
+
+<a id="customization"></a>
+## Customization
+
+* `force` (Optional) This field works as a flag and triggers force customization when `true` during an update 
+(`terraform apply`) every time. It never complains about a change in statefile. It can be used when guest customization
+is needed after a NIC change and then set back to `false`. **Note** this setting will cause a VM reboot and will not
+have effect when `power_on` field is set to `false`.
```

Comment:
As this scenario is stretching the boundaries of who Terraform is used, it would be great to have a workflow example representing the use case from beginning to end (though with HCL snippets showing only the actual parts - no need to show everything). I suggest adding it in this section as opposed to the top. For example, show one snippet with old settings, then how you change the network settings, set force, run apply and set force back to false. Assuming I got everything right... :)

Reply:
Added workflow example.